doc/source/quickstart.rst

```diff
@@ -34,6 +34,8 @@ Once that's all set, you install streamparse using ``pip``::
 
     > pip install streamparse
 
+Note that installing streamparse depends on python-dev libs, hence make
+sure that your distro's python-dev package are also installed.
```

Comment:
We don't build any C extensions or anything, so I don't see why this would be true.